libmcount/internal.h

```diff
@@ -53,6 +53,8 @@ struct filter_control {
 	uint16_t saved_depth;
 	uint64_t time;
 	uint64_t saved_time;
+	unsigned size;
+	unsigned saved_size;
```

Comment:
Once we check it in the entry path, I think we don't need to save it here.

Reply:
Thanks for comments. I update what you advised, so please check them.

Reply:
Don't you need to save it when triggering?

Reply:
You're right, a trigger can change the size filter setting so it needs to remember old filter data.  That means we cannot simply check the size in the entry path to drop them early. Hmm..